benchmark.exs

```diff
@@ -0,0 +1,19 @@
+# Run with `mix run benchmark.exs` in the console
+
+Benchee.run(
+  %{
+    "v1" => fn {expression, context} -> Expression.evaluate(expression, context) end,
+    "v2" => fn {expression, context} -> Expression.V2.eval(expression, Expression.V2.Context.new(context)) end
+  },
+  inputs: %{
+    "simple" => {
+      "Hello @contact.name, you were born in @(YEAR(contact.birthday))",
+      %{"contact" => %{"name" => "mary", "birthday" => ~U[1920-02-02T00:00:00+0000]}}
+    },
+    "map" => {~S|hi @map(0..10, &([&1, concatenate("Button ", &1)])) there|, %{}},
+    "if" =>
+      {"yebo @if(something.false, 1, contact.bar) yes",
+       %{"something" => %{"false" => false}, "contact" => %{}}},
+    "arithmetic" => {"3 * (5 + 2) = @(3 * (5 + 2))", %{}}
+  }
+)
```

Comment:
I don't think it's a huge problem but did you find out why V2 is slower?

Reply:
Not yet but I'm not too concerned about that, I think understandable code is a good enough reason to take the performance hit.